### PR TITLE
Rework customer authentication descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks the customer authentication descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/paths/authentication-tokens@{token}.yaml [hunk 1]
```diff
@@ -14,14 +14,14 @@ get:
   operationId: GetAuthenticationTokenVerification
   x-sdk-operation-name: verify
   description: |
```
Comment (vykhrystiuk) [inline openapi/paths/authentication-tokens@{token}.yaml:16]:
```suggestion
  description: |-
```

File: openapi/paths/authentication-tokens@{token}.yaml [hunk 2]
```diff
@@ -53,14 +53,14 @@ delete:
   operationId: DeleteAuthenticationToken
   x-sdk-operation-name: logout
   description: |
```
Comment (vykhrystiuk) [inline openapi/paths/authentication-tokens@{token}.yaml:55]:
```suggestion
  description: |-
```

File: openapi/paths/credentials.yaml [hunk 3]
```diff
@@ -7,13 +7,13 @@ get:
   operationId: GetCredentialCollection
   x-sdk-operation-name: getAllCredentials
   description: |
```
Comment (vykhrystiuk) [inline openapi/paths/credentials.yaml:9]:
```suggestion
  description: |-
```

File: openapi/paths/credentials.yaml [hunk 4]
```diff
@@ -51,7 +51,7 @@ post:
   operationId: PostCredential
   x-sdk-operation-name: createCredential
   description: |
```
Comment (vykhrystiuk) [inline openapi/paths/credentials.yaml:53]:
```suggestion
  description: |-
```

File: openapi/paths/credentials@{id}.yaml [hunk 5]
```diff
@@ -9,10 +9,10 @@ get:
   operationId: GetCredential
   x-sdk-operation-name: getCredential
   description: |
```
Comment (vykhrystiuk) [inline openapi/paths/credentials@{id}.yaml:11]:
```suggestion
  description: |-
```

File: openapi/paths/credentials@{id}.yaml [hunk 6]
```diff
@@ -39,11 +39,11 @@ put:
     - Core
   tags:
     - Customer authentication
-  summary: Create or update a credential with predefined ID
+  summary: Create or update a credential
   operationId: PutCredential
   x-sdk-operation-name: updateCredential
   description: |
```
Comment (vykhrystiuk) [inline openapi/paths/credentials@{id}.yaml:45]:
```suggestion
  description: |-
```

File: openapi/paths/credentials@{id}.yaml [hunk 7]
```diff
@@ -95,10 +95,10 @@ delete:
   operationId: DeleteCredential
   x-sdk-operation-name: deleteCredential
   description: |
```
Comment (vykhrystiuk) [inline openapi/paths/credentials@{id}.yaml:97]:
```suggestion
  description: |-
```

File: openapi/paths/authentication-options.yaml [hunk 8]
```diff
@@ -40,18 +40,18 @@ put:
   operationId: PutAuthenticationOption
   x-sdk-operation-name: updateAuthOptions
   description: |
```
Comment (vykhrystiuk) [inline openapi/paths/authentication-options.yaml:42]:
```suggestion
  description: |-
```

**Comment by noon-dawg** [comment]
Thanks for the review @vykhrystiuk!

**Review by vykhrystiuk — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**